lib/ImagePrototypeInit.js

```diff
@@ -47,6 +47,19 @@
         this.__locked = false;
     };
 
+    Image.prototype.getMetadata = function() {
+        if (!this.__metadata || this.__metadata === "") {
```

Comment:
We should probably allow users to set metadata to an empty string.
Also, since all the decoders now set 'metadata' to be at least an empty string, I think this check is redundant.